roles/openshift_control_plane/tasks/static.yml

```diff
@@ -29,7 +29,7 @@
     src: "{{ mktemp.stdout }}/{{ item }}"
     edits:
     - key: spec.containers[0].image
-      value: "{{ osm_image }}:{{ openshift_image_tag }}"
+      value: "{{ osm_image }}"
```

Comment:
Yeah, it makes sense to just reference a variable here, the string should be built in defaults.